packages/console/src/cloud/components/Reservation/index.tsx

```diff
@@ -0,0 +1,44 @@
+import type { AdminConsoleKey } from '@logto/phrases';
+import { useTranslation } from 'react-i18next';
+
+import Calendar from '@/assets/images/calendar.svg';
+import Button from '@/components/Button';
+import { buildUrl } from '@/utils/url';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  title: AdminConsoleKey;
+  description: AdminConsoleKey;
+  reservationButtonTitle: AdminConsoleKey;
+};
+
+const Reservation = ({ title, description, reservationButtonTitle }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  return (
+    <div className={styles.reservation}>
+      <div className={styles.reservationInfo}>
+        <Calendar className={styles.reservationIcon} />
+        <div>
+          <div className={styles.reservationTitle}>{t(title)}</div>
+          <div className={styles.reservationDescription}>{t(description)}</div>
+        </div>
+      </div>
+      <Button
+        type="outline"
+        title={reservationButtonTitle}
+        onClick={() => {
+          const bookLink = buildUrl('https://calendly.com/logto/30min', {
```

Comment:
define a const value for calendar urls instead of inline string.

Reply:
updated.